index.bs

```diff
@@ -396,7 +466,23 @@ iterable itself?
 The <dfn method for=FileSystemDirectoryHandle>getFile(|name|, |options|)</dfn> method, when invoked,
 must run these steps:
 
-1. TODO
+1. Let |result| be [=a new promise=].
+1. Run the following steps [=in parallel=]:
+  1. If |options|.{{FileSystemGetFileOptions/create}} is `true`:
+    1. Run <b>[=this=]</b>'s [=FileSystemHandle/entry=]'s [=request permission steps=] given
+       «[&nbsp;"{{FileSystemHandlePermissionDescriptor/writable}}" → `true`&nbsp;]».
+       If that throws an exception, [=reject=] |result| with that exception and abort.
+    1. Let |permissionStatus| be the result of running
+       <b>[=this=]</b>'s [=FileSystemHandle/entry=]'s [=query permission steps=] given
+       «[&nbsp;"{{FileSystemHandlePermissionDescriptor/writable}}" → `true`&nbsp;]».
+  1. Else:
+    1. Let |permissionStatus| be the result of running
+       <b>[=this=]</b>'s [=FileSystemHandle/entry=]'s [=query permission steps=] given
+       «[&nbsp;"{{FileSystemHandlePermissionDescriptor/writable}}" → `false`&nbsp;]».
+  1. If |permissionStatus| is not {{PermissionState/"granted"}},
+     reject |result| with a {{NotAllowedError}} and abort these steps.
```

Comment:
Consistency: "abort" vs. "abort these steps"

Reply:
changed to "abort" throughout.